examples/snippets/filesystem/cloud/gcs_download.py

```diff
@@ -0,0 +1,17 @@
+"""
+An example script to demonstrate the use of the gcs_download function to download file from.
+"""
+
+import tamr_toolbox as tbox
+from google.cloud.client import Client as GcsClient
+
+gcs_client = GcsClient()
```

Comment:
This is not the correct module to import
```suggestion
from google.cloud import storage

gcs_client = storage.Client()
```